.ci/install_debian.sh

```diff
@@ -26,20 +26,15 @@ lsb_dist="$(. /etc/os-release && echo "$ID")"
 dist_version="$(lsb_release -c | cut -d: -f2 | sed s/'^\t'//)"
 echo "lsb_dist: ${lsb_dist}"
 echo "dist_version: ${dist_version}"
-# bullseye is not supported by OpenRobotics' repo, but it has already 
-# the right version of Gazebo in its repo, so we just skip everything
-if [[ ("sid" != "$dist_version" && "bullseye" != "$dist_version" && "bookworm" != "$dist_version" && "trixie" != "$dist_version" && "jammy" != "$dist_version") ]]; then
+# Just a limited amount of distros are supported by OSRF repos, for all the other we use the 
+# gazebo packages in regular repos
+if [[ ("bionic" == "$dist_version" || "focal" == "$dist_version" || "buster" == "$dist_version") ]]; then
     mkdir -p /etc/apt/sources.list.d
     echo deb http://packages.osrfoundation.org/gazebo/$lsb_dist\-stable $dist_version main > /etc/apt/sources.list.d/gazebo-stable.list
     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
     apt-get update
     apt-get install -y libgazebo11-dev
-# See https://github.com/robotology/robotology-superbuild/issues/944
-elif [[ ("bookworm" != "$dist_version") ]]; then
+else
```

Comment:
Sorry @traversaro , it was not clear to me, "debian:testing" and "bookworm" are the same thing?

Reply:
Ok.